Fix spacer block resizing

noisysocks opened the pull request:
Fixes #7755.

The sibling inserter was preventing a user from being able to resize the spacer using the top drag handle.

![spacer-resizing](https://user-images.githubusercontent.com/612155/42434808-cd215156-8397-11e8-971c-001c590726ce.gif)

To test:

1. Insert a spacer block
2. Check that it can be resized using both drag handles
2. Insert an image block
2. Check that it can be resized using both drag handles

mcsf commented:
Thanks for looking into this. Unfortunately, this way, one can't get to the upper sibling inserter when selecting a Spacer block.

P.S.: Edited the PR description to fix the "Fixes #xxxx" against the correct issue.

SuperGeniusZeb commented:
Would it make more sense to just remove the drag handle from the top of the Spacer block? There is not really a need for one on both the top and bottom, and dragging from the top works a little oddly.

noisysocks committed:
Remove top resize handle from spacer block

The top resize handle and the sibling inserter are in the same spot.
Removing the top resize handle is a simple fix.

mcsf commented:
@noisysocks: thanks, though the upper sibling inserter is still inaccessible. :)

noisysocks commented:
🤔 perhaps this is intentional? @jasmussen?

It looks like, in `master`, the sibling inserter **never** shows when you hover over the top of the selected block, unless _Fix Toolbar to Top_ is enabled.

![sibling-inserter](https://user-images.githubusercontent.com/612155/42545422-23d5004c-84fb-11e8-9a58-60b2ef33a583.gif)

jasmussen commented:
This is intentional for the sibling inserter _when a block is selected and that block has no toolbar to cover the sibling inserter_.

It seems this aspect is non optimal when a block has no toolbar, as in the case of the spacer. As we look for a way to fix this let's be sure to not regress this sibling inserter behavior for blocks with toolbars

jasmussen commented:
And by the way I thing it's fine to simply not show the top resizing handle. Your mouse is floating in air anyway, when you use it.

noisysocks commented:
I think then let's leave this as is. We can look at making the sibling inserter appear when the block has no toolbar in a seperate PR. Issue here: https://github.com/WordPress/gutenberg/issues/7884

mcsf reviewed (approved):
Thanks for fixing!
